components/CategoryPage.js

```diff
@@ -65,13 +67,26 @@ var CategoryList = React.createClass({
 var Category = React.createClass({
   render: function() {
     return  (
-      <li className="category_list_item">
-        <div className="category-name">
+    <div>
+      <div className="col-lg-2 col-md-2 col-sm-2 hidden-xs category_option">
           <Link className="category_brand" to={{ pathname: "resources", query: { categoryid: this.props.categoryid } }} >
-            <span>{this.props.name}</span> 
+              <img src={'assets/images/' + this.props.image_path} alt="boohoo" className="img-responsive"/><br />
```

Comment:
Lol, alt should probably be helpful. What do? I mean, the category's name will be right there too.